docs/docs/get_started/configuration.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: Configuration
 id: configuration
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/get_started/configuration.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```

Comment:
I think this page got dropped - I'm looking for a better link

Reply:
:)

Reply:
https://docs.secretless.io/Latest/en/Content/docs/get_started/configuration.htm